config/user_config.go

```diff
@@ -84,9 +84,18 @@ func (cfg *UserConfig) resolve(path string) string {
 	if strings.HasPrefix(path, "~"+string(os.PathSeparator)) {
 		return strings.Replace(path, "~", cfg.Home, 1)
 	}
+	// if using "~/" on Windows
```

Comment:
On Windows it's entirely plausible to write `~/dir` to indicate `C:\\Users\\alice\\dir`, yet `os.PathSeperator` on Windows is `\` so the conditional above doesn't work, but it does if someone writes `~\\dir`